ldap/servers/slapd/plugin.c

```diff
@@ -3366,11 +3366,9 @@ plugin_remove_plugins(struct slapdplugin *plugin_entry, char *plugin_type __attr
                      */
                     return PLUGIN_BUSY;
                 }
-                if (plugin->plg_started) {
-                    Slapi_PBlock *pb = slapi_pblock_new();
-                    plugin_call_one(plugin, SLAPI_PLUGIN_CLOSE_FN, pb);
-                    slapi_pblock_destroy(pb);
-                }
+                Slapi_PBlock *pb = slapi_pblock_new();
+                plugin_call_one(plugin, SLAPI_PLUGIN_CLOSE_FN, pb);
+                slapi_pblock_destroy(pb);
```

Comment:
This change revert the function to the previous status, so it should be safe. Now I have a concern with badly written plugin where the call to the close function expects the call to the start function.

For example [acct_policy close function](https://github.com/389ds/389-ds-base/blob/master/ldap/servers/plugins/acctpolicy/acct_init.c#L217), frees a static config entry that is only initialized during start. it may crash if globalcfg is not initialized to 0.

Reply:
Sounds like we need a new ticket to investigate this...